src/shared/logging/logger.ts

```diff
@@ -45,10 +45,14 @@ export class Logger implements DiagLogger {
         this.updateLogLevel(this._diagLevel);
     }
 
-    public updateLogLevel(logLevel: DiagLogLevel) {
+    public updateLogLevel(logLevel: DiagLogLevel, suppressOverrideMessage?: boolean) {
```

Comment:
```suggestion
    public updateLogLevel(logLevel: DiagLogLevel, suppressOverrideMessage: boolean = true) {
```

Reply:
Just to make sure, this will change the default behaviour. are we good with this?